static/app/components/compactSelect/control.tsx

```diff
@@ -414,6 +408,29 @@ export function Control({
     [selectedOptions]
   );
 
+  const {keyboardProps: optionsWrapKeyboardProps} = useKeyboard({
+    onKeyDown: e => {
+      e.continuePropagation();
+
+      if (!searchRef.current) {
+        return;
+      }
+
+      // When a printable key or backspace is pressed, move focus to the search input
+      // to begin search
+      if (e.key.length === 1 || e.key === 'Backspace') {
```

Comment:
`e.key.length === 1` is an indirect but pretty good way to test for [printable keys](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key). All other control/special keys have `code` values longer than 1.

Reply:
Can you leave this as a comment actually